Parsers/ASimDns/Parsers/ASimDnsDnsMonster.yaml

```diff
@@ -0,0 +1,52 @@
+Parser:
+  Title: DNS activity ASIM parser for DNSmonster
+  Version: '0.1'
+  LastUpdated: Mar 11 2022
+Product:
+  Name: DNSmonster
+Normalization:
+  Schema: Dns
+  Version: '0.1.3'
+References:
+- Title: ASIM DNS Schema
+  Link: https://aka.ms/ASimDnsDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing DNSmonster logs to the ASIM DNS activity normalized schema.
+ParserName: ASimDnsDnsMonster
+ParserQuery: |
+ let rcodes = dynamic(["NOERROR","FORMERR","SERVFAIL","NXDOMAIN","NOTIMP","REFUSED","YXDOMAIN","YXRRSET","NXRRSET","NOTAUTH","NOTZONE","DSOTYPENI"]);
+ let rrcodes = dynamic(["RESV","A","NS","MD","MF","CNAME","SOA","MB","MG","MR","NULL","WKS","PTR","HINFO","MINFO","MX","TXT","RP","AFSDB","X25","ISDN","RT","NSAP","NSAP-PTR","SIG","KEY","PX","GPOS","AAAA","LOC","NXT","EID","NIMLOC","SRV","ATMA","NAPTR","KX","CERT","A6","DNAME","SINK","OPT","APL","DS","SSHFP","IPSECKEY","RRSIG","NSEC","DNSKEY","DHCID","NSEC3","NSEC3PARAM","TLSA","SMIMEA","UNASSIGN","HIP","NINFO","RKEY","TALINK","CDS","CDNSKEY","OPENPGPKEY","CSYNC","ZONEMD","SVCB","HTTPS"]);
+ let opcodes = dynamic(["Query","IQuery","Status","Unassigned","Notify","Update","DNS Stateful Operations"]);
+ let parser=(disabled:bool=false)
+  {
+    dnsmonster_CL | where not(disabled)
+    | where DNS_Response_b
+    | extend DNS_Question = todynamic(DNS_Question_s)
+    | mv-expand DNS_Question
```

Comment:
Using mv-expand means that you will get multiple events, one for each DNS question. 

The DNS schema does not support multiple questions in a single event because after researching it we found that while a valid DNS optoin, it is never really used. Did you encounter a real-world example of multiple questions in a single DNS query?

Reply:
No.  I may have misunderstood the function then.  DNS_Question contains values such as

```
"[
  {
    ""Name"": ""p13n.adobe.io."",
    ""Qtype"": 1,
    ""Qclass"": 1
  }
]"
```

Which I need to use later as DNS_Question.Qtype and DNS_Question.QClass.  mv-expand was a way to break out those values but I can just easily reference as DNS_Question[0].Qtype without the mv-expand.  As you indicated, I have not seen multiple Questions in a single query.